Cargo.toml

```
@@ -9,13 +9,13 @@ exclude = [".github/", "esptool.patch"]
 
 [dependencies]
 critical-section  = "1.1.1"
-esp32-hal = { version = "0.13.0", optional = true, git = "https://github.com/esp-rs/esp-hal", rev = "00fac71" }
-esp32c2-hal = { version = "0.8.0", optional = true, git = "https://github.com/esp-rs/esp-hal", rev = "00fac71" }
-esp32c3-hal = { version = "0.10.0", optional = true, git = "https://github.com/esp-rs/esp-hal", rev = "00fac71" }
-esp32c6-hal = { version = "0.3.0", optional = true, git = "https://github.com/esp-rs/esp-hal", rev = "00fac71" }
-esp32h2-hal = { version = "0.1.0", optional = true, git = "https://github.com/esp-rs/esp-hal", rev = "00fac71" }
-esp32s2-hal = { version = "0.10.0", optional = true, git = "https://github.com/esp-rs/esp-hal", rev = "00fac71" }
-esp32s3-hal = { version = "0.10.0", optional = true, git = "https://github.com/esp-rs/esp-hal", rev = "00fac71" }
+esp32-hal = { version = "0.14.0", optional = true,  git = "https://github.com/esp-rs/esp-hal", rev = "7866896b708a634d9f523f915dff06e983e8ec5d" }
```

```suggestion
esp32-hal = { version = "0.14.0", optional = true, git = "https://github.com/esp-rs/esp-hal", rev = "7866896b708a634d9f523f915dff06e983e8ec5d" }
```

Once we release our next version, would there be any reason to keep using the revision?

---

Nope! Not sure when that will be though, if its soon I'll wait on this PR and change to the released version :)

---

Hoping to cut new releases today, so may want to hold off on merging this.